com.exok.Celeste64.yml

```
@@ -0,0 +1,54 @@
+id: com.exok.Celeste64
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  arch:
+    x86_64:
+      env:
+        RUNTIME: "linux-x64"
+# support for aarm64 will be added in the future
+#    aarch64:
+#      env:
+#        RUNTIME: "linux-arm64"
+  prepend-pkg-config-path: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig"
```

```suggestion
  prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
```

prepend means you don't need to add the whole of existing paths